samples/policy.ts

```diff
@@ -0,0 +1,44 @@
+import * as common from './common';
+import * as nodeApi from 'azure-devops-node-api';
+
+import * as PolicyApi from 'azure-devops-node-api/PolicyApi';
+import * as PolicyInterfaces from 'azure-devops-node-api/interfaces/PolicyInterfaces';
+import * as VSSInterfaces from 'azure-devops-node-api/interfaces/common/VSSInterfaces';
+
+export async function run() {
+    let webApi: nodeApi.WebApi = await common.getWebApi();
+    let policyApiObject: PolicyApi.IPolicyApi = await webApi.getPolicyApi();
+
+    common.banner('Policy Samples');
+    let project: string = common.getProject();
```

Comment:
const

Reply:
We can discuss this, it may be better to create a project with a random name so that we can show an example of how to create/read/update/delete policies. This sample will rely on what data is in the user's account.

Reply:
I agree. This would also be useful across other samples as well, so it might be helpful to have a script that runs before running any samples which creates a project and one at the end to tear it down. 

The only concern I've had with this is that it seems to me like it takes some time to spin up a project (thus the method in coreApi being called queueCreateProject instead of just createProject) - do you know how significant this is? I wouldn't want to do it if it was going to take a minute to spin up the project

Reply:
That's a great point. The project creation is done on a Job Agent so the request is queued up, hence the name. So when the HTTP call returns it gives you a status that may not be complete. I think we may need to poll the URL in the response object until it's complete? It's worth examining the response and trying to deduce what we need to do. We may also be able to query for the project by id and check the status?

I agree we don't want to do this a ton but it seems fine to create 1 test project per run of samples(and delete it at the end).

Reply:
Ok, I'll play around with it and see if I can get it working, I definitely agree that this would be useful though. Honestly, having the project creation as a sample would be useful on its own

Reply:
Agreed! Worth the investment to figure out how to make this work best.